learn/core_concepts/relevancy.md

```diff
@@ -16,28 +16,29 @@ The order in which they are applied has a significant impact on the search resul
 
 By default, ranking rules are executed in the following order:
 
-**1. Typo**
-Results are sorted by **increasing number of typos**: find documents that match query terms with fewer typos first.
-
-**2. Words**
+**1. Words**
 Results are sorted by **decreasing number of matched query terms** in each matching document: find documents that contain more occurrences of the query terms first.
 
-::: warning
+::: note
+
+Be aware that the `words` rule works from right to left. Therefore, the order of the query string impacts the order of results.
+
+For example, if someone were to search `batman dark knight`, then the `words` rule would rank documents containing all three terms first, documents containing only `batman` and `dark` second, and documents containing only `batman` third. All other documents would be ranked last.
```

Comment:
> All other documents would be ranked last.

Do you mean documents that contain only the word `batman` but with typos? I prefer that we remove this sentence. Only the documents that contain `batman` will be ranked last.